### PR TITLE
Dependabot updates December

### PR DESCRIPTION
**Pull request by vidvidvid**
## Overview

**What features/fixes does this PR include?**

- dids #950 
- typescript https://github.com/MetaFam/TheGame/pull/978
-  storybook/addon-links https://github.com/MetaFam/TheGame/pull/970
- opensea-js #969 
- jsdom #954 
- eslint-config-airbnb #955 

Hopefully fixed the cases where the dependabot pr only increased the versions in package.lock - i updated the the versioning strategy of the dependabot so that the package.json's versions also increase:
`versioning-strategy: increase`
[source](https://stackoverflow.com/questions/60201543/dependabot-only-updates-lock-file)

paired-with: @firattale

### CODE REVIEW

**Comment by vidvidvid** [comment]
will team up on this with @firattale

**Comment by firattale** [comment]
I will start this today.

**Comment by vidvidvid** [comment]
close & open for redeploy

**Review by dan13ram — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -76,11 +76,11 @@
     "lerna": "4.0.0",
     "lint-staged": "11.0.0",
     "prettier": "2.2.1",
-    "react": "16.14.0",
-    "react-dom": "16.14.0",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```
Comment (alalonde) [inline package.json:80]: Will [this](https://reactjs.org/blog/2020/10/20/react-v17.html#changes-to-event-delegation) impact anything?

**Review by alalonde — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (firattale) [reply package.json:80]: As I know @vidvidvid tried and it didnt create any problem.

Comment (vidvidvid) [reply package.json:80]: syntax wise it shouldn't create any problem or change.

Comment (vidvidvid) [reply package.json:80]: so if everything else looks good, i'd say we're ready to merge.

**Pull request merged**